src/legacy/server/i18n/index.js

```diff
@@ -20,49 +20,66 @@
 import { resolve } from 'path';
 import globby from 'globby';
 import { i18n, i18nLoader } from '@kbn/i18n';
-
+import { readFile } from 'fs';
 import { fromRoot } from '../../utils';
+import { promisify } from 'util';
+
+const readFileAsync = promisify(readFile);
+
+export async function getTranslationPaths({ cwd, glob }) {
+  const entries = await globby(glob, { cwd });
+
+  const configFiles = await Promise.all(
+    entries
+      .map(entry => resolve(cwd, entry))
+      .map(async configFilePath => ({
+        path: configFilePath,
+        content: await readFileAsync(configFilePath, 'utf8')
+      }))
+  );
+
+  return configFiles
+    .map(({ content, path }) => {
+      try {
+        return JSON.parse(content);
+      } catch (err) {
+        throw new Error(`Failed to parse .i18nrc.json file at ${path}`);
+      }
+    })
+    .reduce((acc, configFile) => acc.concat(...configFile.translations), [])
+    .map(translationPath => resolve(cwd, translationPath));
```

Comment:
I kept this function `getTranslationPaths` indifferent of what the locale is, it just grabs all the locales, then the implementation does the filtering, in case we want to change this behavior in the future (like supporting multiple locales per kibana run).